dotcom-rendering/src/web/components/Page.tsx

```diff
@@ -1,36 +1,44 @@
-import React, { StrictMode } from 'react';
-import { EmotionCache } from '@emotion/cache';
-import { CacheProvider, Global, css } from '@emotion/react';
+import { StrictMode } from 'react';
+import { Global, css } from '@emotion/react';
 import { focusHalo } from '@guardian/source-foundations';
+import { ArticleDesign } from '@guardian/libs';
+import { SkipTo } from './SkipTo';
+import { DecideLayout } from '../layouts/DecideLayout';
 
 type Props = {
-	children: React.ReactNode;
-	cache: EmotionCache;
+	CAPI: CAPIType;
+	NAV: NavType;
+	format: ArticleFormat;
 };
 
 /**
  * @description
- * Page is a high level wrapper for pages on Dotcom. Sets strict mode, some globals
- * and the Emotion cache
+ * Page is a high level wrapper for pages on Dotcom. Sets strict mode and some globals
  *
- * @param {ReactNode} children - What gets rendered on the page
- * @param {EmotionCache} cache - Provides the Emotion cache
+ * @param {CAPIType} CAPI - The article JSON data
+ * @param {NAVType} NAV - The article JSON data
+ * @param {ArticleFormat} format - The format model for the article
  * */
-export const Page = ({ children, cache }: Props) => {
+export const Page = ({ CAPI, NAV, format }: Props) => {
 	return (
-		<CacheProvider value={cache}>
-			<StrictMode>
-				<Global
-					styles={css`
-						/* Crude but effective mechanism. Specific components may need to improve on this behaviour. */
-						/* The not(.src...) selector is to work with Source's FocusStyleManager. */
-						*:focus {
-							${focusHalo}
-						}
-					`}
-				/>
-				{children}
-			</StrictMode>
-		</CacheProvider>
+		<StrictMode>
+			<Global
+				styles={css`
+					/* Crude but effective mechanism. Specific components may need to improve on this behaviour. */
+					/* The not(.src...) selector is to work with Source's FocusStyleManager. */
+					*:focus {
+						${focusHalo}
+					}
+				`}
+			/>
+			<SkipTo id="maincontent" label="Skip to main content" />
+			<SkipTo id="navigation" label="Skip to navigation" />
+			{format.design === ArticleDesign.LiveBlog ||
+				(format.design === ArticleDesign.DeadBlog && (
+					<SkipTo id="keyevents" label="Skip to key events" />
+				))}
```

Comment:
I'm not super familiar with the JS syntax or the requirement here, just double checking: Is this equivalent to the previous syntax?

`if (
		CAPI.format.design === 'LiveBlogDesign' ||
		CAPI.format.design === 'DeadBlogDesign'
	) {
		skipToKeyEvents = renderToString(
			<SkipTo id="keyevents" label="Skip to key events" />,
		);
	}`

Reply:
Basically, yes. The conditional is easier to read in classic javascript vs. JSX and it doesn't help that prettier has added those brackets in such a weird way 🤔